OnePlus/OP6/res/values/config.xml

```diff
@@ -64,7 +64,13 @@
     <bool name="config_wifi_background_scan_support">true</bool>
     <bool name="config_wifi_dual_band_support">true</bool>
 
-    <!--<string name="config_dozeComponent">com.android.systemui/com.android.systemui.doze.DozeService</string>
+    <string name="config_dozeComponent">com.android.systemui/com.android.systemui.doze.DozeService</string>
     <bool name="config_dozeAlwaysOnDisplayAvailable">true</bool>
-    <bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>-->
+    <bool name="config_displayBlanksAfterDoze">true</bool>
+    <bool name="config_powerDecoupleInteractiveModeFromDisplay">false</bool>
+    <bool name="config_enableAutoPowerModes">true</bool>
+    <bool name="config_powerDecoupleAutoSuspendModeFromDisplay">true</bool>
+    <bool name="config_supportDoubleTapWake">true</bool>
```

Comment:
That's false in stock framework-res

Reply:
I dunno if the value it's still false, in an OP6 with latest update the bool for the DT2W seems at work since the feature it's available.